content/arduino-cloud/03.cloud-interface/02.variables/variables.md

```diff
@@ -265,8 +265,10 @@ To read the location values, we can use the following method `Location coordinat
 
 | Property  | Type    | Read value        | Set value                   |
 | --------- | ------- | ----------------- | --------------------------- |
-| Latitude  | `float` | `coordinates.lat` | This variable is ready only |
-| Longitude | `float` | `coordinates.lon` | This variable is ready only |
+| Latitude  | `float` | `coordinates.lat` | `latitude = latitudeValue;` |
+| Longitude | `float` | `coordinates.lon` | `longitude = longitudeValue;` |
```

Comment:
The information you propose to add under the "Set value" column is meaningless.

I don't think it makes sense to try to communicate how to set the value via the column. I suggest you instead direct the reader to the prose content you added after the table:

```suggestion
| Latitude  | `float` | `coordinates.lat` | See information below |
| Longitude | `float` | `coordinates.lon` | See information below |
```